packages/expo-media-library/src/MediaLibrary.ts

```diff
@@ -93,6 +95,11 @@ export type PagedInfo<T> = {
   totalCount: number;
 };
 
+export type PermissionsResult = {
+  status: 'granted' | 'denied' | 'undetermined';
```

Comment:
Maybe extract that to some Enum and `export ` as well. That would allow users to use library provided const and not `if (someResult === 'granted`)` 🤔 ❓